src/wazuh_db/schema_global_upgrade_v4.sql

```diff
@@ -0,0 +1,34 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ *
+ * November, 2021.
+ *
+ * This program is a free software, you can redistribute it
+ * and/or modify it under the terms of GPLv2.
+*/
+
+CREATE TABLE IF NOT EXISTS _belongs (
+    id_agent INTEGER REFERENCES agent (id) ON DELETE CASCADE,
+    id_group INTEGER REFERENCES `group`(id),
+    PRIMARY KEY (id_agent, id_group)
+);
+
+CREATE INDEX IF NOT EXISTS belongs_id_agent ON belongs (id_agent);
+CREATE INDEX IF NOT EXISTS belongs_id_group ON belongs (id_group);
+
+BEGIN;
+
+INSERT INTO _belongs (id_agent, id_group) SELECT id_agent, id_group FROM belongs WHERE id_agent IN (SELECT id FROM agent) AND id_group IN (SELECT id FROM `group`);
```

Comment:
I like this insert in which we verify the agent and group existence.

Reply:
Thanks! It is an extension of the query used in the previous upgrade schema, adding the `id_group` part